cognitive/src/test/python/synapsemltest/services/langchain/test_LangchainTransform.py

```diff
@@ -95,6 +95,44 @@ def test_langchainTransform(self):
         # column has the expected result.
         self._assert_chain_output(self.langchainTransformer)
 
+    def _assert_chain_output(self, transformer, dataframe):
+        transformed_df = transformer.transform(dataframe)
+        input_col_values = [row.technology for row in transformed_df.collect()]
+        output_col_values = [row.copied_technology for row in transformed_df.collect()]
```

Comment:
nit: this test will run twice as fast if you collect once and then pull out the info from that collected df

Reply:
modified this according to the suggestion